File: Java/AutomationFramework/src/com/app/UITests/DashboardPageTests/DashboardMenuListTest.java
package com.app.UITests.DashboardPageTests;

import com.app.pages.DashboardPage;
import org.testng.Assert;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

import java.util.Arrays;
import java.util.List;

public class DashboardMenuListTest extends BaseDashboardTest{
    DashboardPage Dashboard_Page;

    @BeforeMethod
    public void SetUp()
    {
        Dashboard_Page = LoginUser();
    }

    @Test
    public void MenuListTest()
    {
        List<String> list = Arrays.asList( "Admin", "PIM", "Leave", "Time", "Recruitment", "My Info", "Performance", "Dashboard", "Directory", "Maintenance", "Claim", "Buzz" );

        List<String> menuList = Dashboard_Page.GetMenuList();

        Assert.assertEquals(menuList, list);
    }
}

File: DotNet/Framework/AutomationFramework/Tests.UITests/DashboardPageTests/DashboardMenuListTest.cs
using NUnit.Framework;
using Pages;

namespace Tests.UITests.DashboardPageTests
{
    [TestFixture]
    public class DashboardMenuListTest : BaseTestFixtures
    {
        DashboardPage Dashboard_Page;

        [SetUp]
        public void SetUp()
        {
            Dashboard_Page = LoginUser();
        }

        [Test]
        public void MenuListTest()
        {
            List<string> list = new List<string> { "Admin", "PIM", "Leave", "Time", "Recruitment", "Performance", "Dashboard", "Directory", "Maintenance", "Claim", "Buzz", "My Info" };
            
            Dashboard_Page.GetMenuList(out List<string> menuList);

            Assert.That(list, Is.EquivalentTo(menuList));
        }
    }
}
